packages/api-server/src/methods/error.ts

```diff
@@ -32,6 +32,19 @@ export class InvalidParamsError extends RpcError {
   constructor(message: string) {
     super(INVALID_PARAMS, message);
   }
+
+  padErrorContext(context: string): InvalidParamsError {
+    const msgs = this.message.split(/(invalid argument .: )/);
+    // [ '', 'invalid argument <number>: ', 'message' ]
+    if (msgs.length !== 3) {
+      throw new Error(
```

Comment:
Which is better to throw a new error or return the original error?

Reply:
let's throw original one for robustness